src/Controller/FilesController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * The FilesController handles the endpoint where to send files to.
+ *
+ * @license EUPL <https://github.com/ConductionNL/contactcatalogus/blob/master/LICENSE.md>
+ *
+ * @category Controller
+ */
+
+namespace CommonGateway\CoreBundle\Controller;
+
+// use CommonGateway\CoreBundle\Service\UploadService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+use function Safe\json_encode;
+
+class FilesController extends AbstractController
+{
+
+    // /**
+    // * @var UploadService The UploadService.
+    // */
+    // private UploadService $uploadService;
+
+    /**
+     * The constructor sets al needed variables.
+     *
+     * @param UploadService $UploadService.
+     */
+    public function __construct(
+        // UploadService $UploadService
+    ) {
+        // $this->UploadService = $UploadService;
+    }//end __construct()
+
+    /**
+     * Provides a files endpoint.
+     *
+     * @Route("/admin/files-upload", methods={"POST"})
```

Comment:
small remark, but usually I would expect file to be singular in a file upload.

Reply:
Will correct this